src/pages/dialog/components/MessageItem.module.css

```diff
@@ -0,0 +1,53 @@
+img {
+	border-radius: 50%;
+	width: 50px;
+	height: 50px;
+}
+
+
+.YourMessage {
```

Comment:
For fix design change this to 
max-width: 40vw;
display: flex;
margin-left: 5vw;